Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml

```diff
@@ -3168,7 +3177,7 @@ tasks:
                       iscontext: true
                     right:
                       value:
-                        simple: PaloAltoNetworksXDR.OriginalAlert._all_events.causality_actor_primary_normalized_user.username
+                        simple: inputs.Username
```

Comment:
didn't see this as a input
same as here

Reply:
this change was made by the 'format' command. Reverted.